lib/string.js

```diff
@@ -72,6 +72,16 @@ internals.String.prototype._base = function (value, state, options) {
         }
     }
 
+    if (typeof value === 'string' &&
+        this._inner.replacements &&
+        options.convert) {
```

Comment:
I'm still thinking about it, convert is documented as a type coercing property, this is not the case here, I'm not sure we need it. (though we could argue about lowercase and others)